test/e2e/e2e_test.go

```diff
@@ -140,23 +279,200 @@ var _ = ginkgo.Describe("Kueue visibility server", func() {
 			ginkgo.By("Await for pods to be running", func() {
 				gomega.Eventually(func() int {
 					createdJob := &batchv1.Job{}
-					gomega.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(sampleJob1), createdJob)).Should(gomega.Succeed())
+					gomega.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(blockingJob), createdJob)).Should(gomega.Succeed())
 					return int(*createdJob.Status.Ready)
 				}, util.Timeout, util.Interval).Should(gomega.Equal(1))
 			})
 
 			ginkgo.By("Terminate execution of the first workload to release the quota", func() {
-				gomega.Expect(util.DeleteAllPodsInNamespace(ctx, k8sClient, ns)).Should(gomega.Succeed())
+				gomega.Expect(util.DeleteAllPodsInNamespace(ctx, k8sClient, nsA)).Should(gomega.Succeed())
 			})
 
 			ginkgo.By("Verify there are zero pending workloads, after the second workload is admitted", func() {
 				gomega.Eventually(func() int {
-					info, err := visibilityClient.ClusterQueues().GetPendingWorkloadsSummary(ctx, clusterQueue.Name, metav1.GetOptions{})
+					info, err := visibilityClient.LocalQueues(nsA.Name).GetPendingWorkloadsSummary(ctx, localQueueA.Name, metav1.GetOptions{})
 					gomega.Expect(err).NotTo(gomega.HaveOccurred())
 					return len(info.Items)
 				}, util.Timeout, util.Interval).Should(gomega.Equal(0))
 			})
 		})
+
+		ginkgo.It("Should allow fetching information about position of pending workloads from different LocalQueues", func() {
```

Comment:
@PBundyra aren't we missing RBAC yamls here for LQs? I'm wondering why the e2e test works in this case.

Reply:
We definitely are, thanks for pointing that out, yet had to test it. 
>> I'm wondering why the e2e test works in this case.

I guess the e2e tests client has access to every resource